scoped/impls.go

```diff
@@ -0,0 +1,2570 @@
+// Code generated by go generate; DO NOT EDIT.
+
+package scoped
```

Comment:
pls rename that file `impls.gen.go` so we follow what we started in the IPv6 to ASN package.